api-definitions/db_positions/api.yaml

```
@@ -0,0 +1,296 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Debtor Positions API
+  contact:
+    name: PagoPA
+    url: https://www.pagopa.gov.it/
+  x-logo:
+    url: https://www.pagopa.gov.it/assets/images/pagopa-logo.png
+  description: Specification for Debtor Positions API
+host: api.io.italia.it.TOBEDEFINE
+basePath: "/api/v1"
+schemes:
+  - https
+security:
+  - ApiKey: []
+paths:
+  "/payment-notices/{organizationfiscalcode}/payment-notices/{paymentnoticenumber}":
+    post:
+      operationId: createPosition
+      summary: |
+        The PA creates a Debt Position
+      parameters:
+        - $ref: "#/parameters/OrganizationFiscalCode"
+        - $ref: "#/parameters/PaymentNoticeNumber"
+        - name: payment_position
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/PaymentPosition"
+      responses:
+        "201":
+          description: Request created.
+          schema:
+            $ref: "#/definitions/PaymentPosition"
+        "401":
+          description: Wrong or missing function key.
+        "409":
+          description: "Conflict: pending request found."
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+    get:
+      operationId: getPosition
+      summary: |
+        The PA gets the details of a specific debt position.
+      parameters:
+        - $ref: "#/parameters/OrganizationFiscalCode"
+        - $ref: "#/parameters/PaymentNoticeNumber"        
+      responses:
+        "200":
+          description: payment position details.
+          schema:
+            $ref: "#/definitions/PaymentPosition"
+        "401":
+          description: Wrong or missing function key.
+        "404":
+          description: No payment found.
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+    delete:
+      operationId: delPosition
+      summary: |
+        The PA can only cancel its own debt positions, or rather the entities it manages.
+      parameters:
+        - $ref: "#/parameters/OrganizationFiscalCode"
+        - $ref: "#/parameters/PaymentNoticeNumber"        
+      responses:
+        "200":
+          description: payment position details.
+          schema:
+            $ref: "#/definitions/PaymentPosition"
+        "401":
+          description: Wrong or missing function key.
+        "404":
+          description: No payment position found.
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+    put:
+      operationId: updatePosition
+      summary: |
+        The PA update a Debt Position
```

```suggestion
       Update a Debt Position
```

there's no need to repeat the actor who's calling the API